internal/bundle/bind_resource_test.go

```diff
@@ -101,12 +101,12 @@ func TestAccAbortBind(t *testing.T) {
 		destroyBundle(t, ctx, bundleRoot)
 	})
 
-	t.Setenv("BUNDLE_ROOT", bundleRoot)
-	c := internal.NewCobraTestRunner(t, "bundle", "deployment", "bind", "foo", fmt.Sprint(jobId))
-
-	// Simulate user aborting the bind. This is done by not providing any input to the prompt in non-interactive mode.
-	_, _, err = c.Run()
-	require.ErrorContains(t, err, "failed to bind the resource")
+	// Bind should fail because prompting is not possible.
+	stdout, stderr, err := blackBoxRun(t, bundleRoot, "bundle", "deployment", "bind", "foo", fmt.Sprint(jobId))
```

Comment:
The original test fails because it implicitly relied on the stdin not being passed to the cobra test runner, which caused an error here because of `EOF` on stdin. 

The test is now improved, and the error message is more prescriptive to use the `--auto-approve` flag. Testing the prompting library itself is out of scope for this test.